### DISCUSSION TITLE
What are those messages?

### DISCUSSION THREAD

**Discussion by kod-man** [vercel/next.js#56866] [App Router]
<img width="863" alt="Screenshot 2023-10-14 at 14 50 56" src="https://github.com/vercel/next.js/assets/54946794/8912cd76-9124-4bd1-ace1-b4d14f97617c">

**Comment by icyJoseph** [comment]
PWA Service worker?

And the `__nextjs_original` I think is result of some error communicating with the server, and that itself can be caused by a lot of things.
